mantidimaging/gui/windows/main/model.py

```diff
@@ -160,3 +160,7 @@ def do_remove_stack(self, stack_uuid: uuid.UUID) -> None:
     @property
     def have_active_stacks(self) -> bool:
         return len(self.active_stacks) > 0
+
+    def add_log_to_sample(self, stack_name: str, log_file: str):
+        stack = self.get_stack_by_name(stack_name).widget()  # type: ignore
+        stack.presenter.images.log_file = loader.load_log(log_file)
```

Comment:
Can we also add a `QMessgeBox` to say `Log loaded successfully` (or show the error as `raise_if_angle_missing` raises a runtime error on image/angle mismatch)?

Currently it just kind of closes and because we can't see the logs from anywhere else it's confusing if anything happened